packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -32,17 +32,17 @@ export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBox
           }
         },
       hasInput && {
-        borderColor: palette.neutralSecondary
+        borderColor: semanticColors.smallInputBorder
```

Comment:
this slot is meant for things like checkboxes
you should use one of the inputBorders here, like the focused one perhaps (theoretically it should match whatever textbox is doing)